lib/src/Uncertainty/Distribution/SquaredNormal.cxx

```diff
@@ -22,47 +22,47 @@
 #include <cmath>
 
 #include "openturns/SquaredNormal.hxx"
-#include "openturns/Distribution.hxx"
-#include "openturns/ChiSquare.hxx"
+//#include "openturns/Distribution.hxx"
+//#include "openturns/ChiSquare.hxx"
 #include "openturns/SpecFunc.hxx"
-#include "openturns/Log.hxx"
-#include "openturns/OSS.hxx"
+//#include "openturns/Log.hxx"
+//#include "openturns/OSS.hxx"
 #include "openturns/DistFunc.hxx"
 #include "openturns/PersistentObjectFactory.hxx"
-#include "openturns/Matrix.hxx"
-#include "openturns/MatrixImplementation.hxx"
-#include "openturns/IdentityMatrix.hxx"
+//#include "openturns/Matrix.hxx"
+//#include "openturns/MatrixImplementation.hxx"
+//#include "openturns/IdentityMatrix.hxx"
 //#include "openturns/NormalCopula.hxx"
-#include "openturns/ResourceMap.hxx"
-#include "openturns/RandomGenerator.hxx"
-#include "openturns/GaussKronrodRule.hxx"
+//#include "openturns/ResourceMap.hxx"
+//#include "openturns/RandomGenerator.hxx"
+//#include "openturns/GaussKronrodRule.hxx"
```

Comment:
you can remove these instead of commenting

Reply:
yeah, I forgot step 2